### ISSUE TITLE
Display conversations all in one list

### ISSUE THREAD

**Issue by L12C** [vector-im/riotX-android#650]
(As an option) add the possibility to show all rooms and conversations in a single list sorted by the time of the latest message (like eg. Signal does) as a start screen, rather than having three tabs for unread, conversations and rooms.

Alternatively, if this is too much work, use the same Components from those tabs but put them one under the other to have a more compact view, so it just takes one tap after starting the app to open one of your recent chats

**Comment by bmarty** [comment]
This is currently discussed with the internal design team.

**Comment by progserega** [comment]
Some times ago my coworker ask me - where her rooms. After login and initial sinc he was not see rooms with more people (groups rooms).
I do not understand he. But after some quiestions I understand problem - He was not understand, that rooms was on other tab... 

May be for new account (first setup element) by default need set "show rooms and personal chats on one tab". But in settings save options:
1. show rooms and personal chats on two tabs.
1. show "home tab" with "new events"

And may be add "use home tab as main tab and show it first" - for advanced users (now it show on end of tabs list and do not show first if open app).

**Comment by jwsp1** [comment]
> This is currently discussed with the internal design team.

I don't understand why waiting for design team is necessary, shouldn't this just remove the tab bar at the bottom?

Element has the experimental option to show a dedicated tab for unread messenges (3 Tabs, like it used to be in riot).
I'd welcome to enhance this setting in order to let the user simply decide between:

- 1 Tab: all together (no tab shown at all, of course)
- 2 Tabs: DM | Rooms 
- 3 Tabs: Home (unread) | DM | Rooms

**Comment by kojid0** [comment]
> This is currently discussed with the internal design team.

What about the design implementation in [schildichat](https://github.com/SchildiChat/schildichat-android)? 
Maybe @SpiritCroc could do a PR for upstream Element?

**Comment by aniseeo** [comment]
Oh my god, this things discussed a long time ago but still not implement. I just don't understand why? just give us an opting no need to do by default.

**Comment by githuberyt** [comment]
+1

**Comment by jwsp1** [comment]
> Oh my god, this things discussed a long time ago but still not implement. I just don't understand why? just give us an opting no need to do by default.

The design team needs to make up their mind about this, I guess. But it seems to have very low priority or none of them were pinged, idk.

**Comment by AnisTaluqdar** [comment]
It can be a very low priority for dev but it is very important for simplicity. For most of the people who don't use the room for them, it feels annoying. Also when you select another space then there has only room no DM. So, it feels like extra garbage.

**Comment by frandavid100** [comment]
Yes, having to click the right tab feels like an extra step now that some rooms are inside spaces. Please, add this option.

**Comment by jwsp1** [comment]
> It can be a very low priority for dev but it is very important for simplicity.

Sholdn't simplicity be high priority?

**Comment by AnisTaluqdar** [comment]
Yes, but this thing is very easy but it's waiting for a long time so I think it's not their high priority.

**Comment by HarHarLinks** [comment]
Related #1241 which is what I would personally prefer over just an option to combine the current 2 tabs.

**Comment by jwsp1** [comment]
> Related #1241 which is what I would personally prefer over just an option to combine the current 2 tabs.

I think this will be implemented with #4010

**Comment by enz0nun3s15** [comment]
The optimal solution would be asking which interface to use when the app is initialized for the first time. IIRC Gmail does something similar when the user is configuring the inbox for the first time, displaying cute previews:

![image](https://user-images.githubusercontent.com/92552606/147241786-98835015-c689-448a-9f8d-7afa2c4c00c5.png)

**Comment by kojid0** [comment]
> The optimal solution would be asking which interface to use when the app is initialized for the first time. IIRC Gmail does something similar when the user is configuring the inbox for the first time, displaying cute previews:

I like the idea. But IMO it is important that users can use the app as quickly as possible after downloading it. This means that the number of questions at the first start should be reduced to a minimum.

I came up with the idea to offer a tour on startup: #4808

**Comment by AnisTaluqdar** [comment]
Switch between space and home it very annoying having multi tab instead of one tab like this propose.

As example you want to go space there have no DM so that tab is empty as also when you cam back home your room space is empty.

**Comment by nathanael-h** [comment]
Hi @bmarty I opened and closed the issue #6442 because it was a duplicate of this one. Actually some people I recommended to use Element find the user experience complicated with spaces and the DM/rooms tabs. A unified list should improve the UX. I read the PR #4010 by @BillCarsonFr  and saw that internal discussions were needed before such a change. As this was a few months ago, could we have an update on that topic? 

I would be really interested in user testing session, or contributing to a PR for adding unified chat list. But before starting something, I'd like to have some fresh information, and guidance maybe.

**Comment by HarHarLinks** [comment]
> What about the design implementation in [schildichat](https://github.com/SchildiChat/schildichat-android)?

schildi's simple mode which includes a unified room list has been very nice for some of my less tech literate folk

**Comment by genofire** [comment]
Looks be solved with the new App Layout.

**Comment by HarHarLinks** [comment]
Agree (but sadly the previous room list has been removed in it's stead...)

**Comment by AnisTaluqdar** [comment]
> Agree (but sadly the previous room list's view filtering low-prio has been removed in it's stead...)

that's really sad
